rocky/reports/templates/partials/export_report_settings.html

```diff
@@ -0,0 +1,127 @@
+{% load i18n %}
+{% load static %}
+{% load report_extra %}
+{% load ooi_extra %}
+{% load component_tags %}
+{% load compress %}
+
+<section>
+    <div>
+        {% if selected_oois and selected_report_types %}
+            <h2>{% translate "Report name" %}</h2>
+            <p>
+                {% blocktranslate trimmed %}
+                    You can name individual reports.
+                {% endblocktranslate %}
+            </p>
+            <form id="generate_report" class="inline" method="post" action="{{ next }}">
+                <table id="report-name-table">
+                    <caption class="visually-hidden">{% translate "Report names:" %}</caption>
+                    <thead>
+                        <tr>
+                            <th scope="col">{% translate "Name" %}</th>
+                            <th scope="col"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for report_name in reports %}
+                            {% with report_id=report_name|slugify %}
+                                <input type="hidden" name="old_report_name" value="{{ report_name }}" />
+                                <tr>
+                                    <td id="text-{{ report_id }}">{{ report_name }}</td>
+                                    <td>
+                                        <input id="{{ report_id }}"
+                                               name="report_name"
+                                               type="hidden"
```

Comment:
ideally, Id want this to be just a text field, and hide it with the JS on load.

Reply:
Since another design will be implemented shortly, that eliminates this case as discussed last friday, I will leave this as is for now.